README.md

```diff
@@ -6,13 +6,15 @@
+<img width="64" src="https://raw.githubusercontent.com/laurent22/joplin/master/Assets/LinuxIcons/256x256.png" align="left" /> **Joplin** is a free, open source note taking and to-do list application available for Windows, Linux, macOS, Android, and iOS. It can easily handle a vast number of notes, which are organised into notebooks. Notes can be searched, copied, tagged, and edited -- even with your favourite text [editor](#External-text-editor).
 
-Notes exported from Evernote via .enex files [can be imported](#importing) into Joplin, including the formatted content (which is converted to Markdown), resources (images, attachments, etc.) and complete metadata (geolocation, updated time, created time, etc.). Plain Markdown files can also be imported.
+Notes can be easily imported from Evernote into Joplin using [.enex files](#importing). Formatted content, resources (images, attachments, etc.), and complete metadata is included in the import, which is stored in [Markdown](#markdown) format. Plain Markdown files can also be imported, increasing flexibility and usability.
 
-The notes can be [synchronised](#synchronisation) with various cloud services including [Nextcloud](https://nextcloud.com/), Dropbox, OneDrive, WebDAV or the file system (for example with a network directory). When synchronising the notes, notebooks, tags and other metadata are saved to plain text files which can be easily inspected, backed up and moved around.
+Another key feature is that notes can be [synchronized](#synchronization) with widely known cloud services such as [Nextcloud](https://nextcloud.com), Dropbox, OneDrive, WebDAV, and simple file systems, which can also facilitate private network storage. When synchronising, notes, notebooks, tags, and other metadata are saved as plain text files with optional [encryption](#Encryption), which can quickly be inspected, copied, moved, and backed up as needed.
 
-The application is available for Windows, Linux, macOS, Android and iOS (the terminal app also works on FreeBSD). A [Web Clipper](https://github.com/laurent22/joplin/blob/master/readme/clipper.md), to save web pages and screenshots from your browser, is also available for [Firefox](https://addons.mozilla.org/firefox/addon/joplin-web-clipper/) and [Chrome](https://chrome.google.com/webstore/detail/joplin-web-clipper/alofnhikmmkdbbbgpnglcpdollgjjfek?hl=en-GB).
+A free [Web Clipper](https://github.com/laurent22/joplin/blob/master/readme/clipper.md) is also available for [Chrome](https://chrome.google.com/webstore/detail/joplin-web-clipper/alofnhikmmkdbbbgpnglcpdollgjjfek?hl=en-GB) to save web pages and screenshots from your browser. The [Firefox](https://addons.mozilla.org/firefox/addon/joplin-web-clipper/) clipper will be available soon. (See [this post](https://discourse.joplinapp.org/t/firefox-web-clipper-not-in-add-ons-store/5809/26)) 
+
+A terminal app is also available for FreeBSD. 
```

Comment:
Moved terminal app info here because it usually only applies to the die-hard *nix guys, who always read everything from top to bottom.